gcc/rust/parse/rust-parse-impl.h

```diff
@@ -14821,7 +14796,7 @@ Parser<ManagedTokenSource>::debug_dump_lex_output (std::ostream &out)
     }
 }
 
-// Parses crate and dumps AST to stderr, recursively.
+// Parses crate and dumps AST to  recursively.
```

Comment:
Please fix (restore `stderr`, I suppose?) that one, and we're good to go!

Reply:
Ah dangit my regex-fu was too powerful. Thanks for catching that